tests/getRoutes.test.js

```diff
@@ -0,0 +1,11 @@
+import request from "supertest";
+const url = "http://localhost:3001";
+jest.setTimeout(30000);
```

Comment:
you can set the 3rd parameter of a test function to set the timeout for that test only, to avoid setting it globally and affecting other tests in unpredictable ways.

```
test("if /hacklist returns an array of hacks in the databse", async () => {
  // test code
}, 30000);
```